scripts/jenkins/srun_dawn_PR.sh

```diff
@@ -14,5 +14,6 @@ srun --job-name=dawn_PR \
      --partition=cscsci \
      --constraint=gpu \
      --account=c14 \
-     $BASEPATH_SCRIPT/dawn_PR.sh
+     $BASEPATH_SCRIPT/dawn_PR.sh -p
```

Comment:
this will change to the above once we merge to have control in jenkins. This cannot be done in the PR setup